### PR TITLE
getting word bank datasets

### PR DESCRIPTION
**Pull request by dcpcamara**
updated version of past commit, now there are two standalone commented functions to download whole World Bank datasets.

### CODE REVIEW

File: R/get_wb_data.R [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#' my_dataset <- datasets[2]
+#' my_data <- get_wb_data(my_dataset)
+#'
+#' my_dataset <- "Environment, Social and Governance (ESG) Data"
+#' my_data <- get_wb_data(my_dataset)
+#' }
+#' @export
+
+get_wb_data <- function(datasets, start_year = 1960, end_year = 2022){
+  require(dplyr)
```
Comment (fccoelho) [inline R/get_wb_data.R:24]: Wouldn't it be better to move these `requires` outside the function? As it is now every time the function is called the requires will be run.

**Review by fccoelho — CHANGES_REQUESTED** [review]

**Comment by fccoelho** [comment]
We  need unit tests in our R-package:

@dcpcamara,  this tutorial may help to get started with unit testing: https://debruine.github.io/tutorials/your-first-r-package-with-unit-tests.html

File: R/get_wb_data.R (hunk 1 above)
Comment (dcpcamara) [reply R/get_wb_data.R]: Done in the newer commit. Instead, all outside functions are listed in the NAMESPACE file.

**Comment by dcpcamara** [comment]
> We need unit tests in our R-package:
> 
> @dcpcamara, this tutorial may help to get started with unit testing: https://debruine.github.io/tutorials/your-first-r-package-with-unit-tests.html

@fccoelho we may have reached a problem. I'm following the instructions on this tutorial, but I can't manage to even check the R package for errors because of its name (an issue that I mentioned in #9). I don't know how to proceed. Maybe we should consider changing the repository name to allow the tests in the tutorial.

**Comment by fccoelho** [comment]
> > We need unit tests in our R-package:
> > @dcpcamara, this tutorial may help to get started with unit testing: https://debruine.github.io/tutorials/your-first-r-package-with-unit-tests.html
> 
> @fccoelho we may have reached a problem. I'm following the instructions on this tutorial, but I can't manage to even check the R package for errors because of its name (an issue that I mentioned in #9). I don't know how to proceed. Maybe we should consider changing the repository name to allow the tests in the tutorial.

See @xmnlab sugestion on the other PR #4

**Pull request merged**